docs-src/go/how-to-use-a-custom-payload-converter-in-go.md

```diff
@@ -0,0 +1,47 @@
+---
+id: how-to-use-a-custom-payload-converter-in-go
+title: How to use a custom Payload Converter in Go
+sidebar_label: Custom Payload Converter
+description: Use a `CompositeDataConverter` to apply custom `PayloadConverter` in a specified order.
+tags:
+  - go
+  - developer-guide
+---
+
+Use a [`CompositeDataConverter`](https://pkg.go.dev/go.temporal.io/sdk/converter#CompositeDataConverter) to apply custom type-specific `PayloadConverter` in a specified order.
+
+The `PayloadConverter` converts bytes to values and back.
+
+`NewCompositeDataConverter` creates new instance of `CompositeDataConverter` from an ordered list of type-specific `PayloadConverters`.
+The following type-specific Payload Converters are available in the Go SDK, listed in the order that they are applied by in the default Data Converter:
+
+- [`NewNilPayloadConverter()`](https://pkg.go.dev/go.temporal.io/sdk/converter#NilPayloadConverter.ToString)
+- [`NewByteSlicePayloadConverter()`](https://pkg.go.dev/go.temporal.io/sdk/converter#ByteSlicePayloadConverter)
+- [`NewProtoJSONPayloadConverter()`](https://pkg.go.dev/go.temporal.io/sdk/converter#ProtoJSONPayloadConverter)
+- [`NewProtoPayloadConverter()`](https://pkg.go.dev/go.temporal.io/sdk/converter#ProtoPayloadConverter)
+- [`NewJSONPayloadConverter()`](https://pkg.go.dev/go.temporal.io/sdk/converter#JSONPayloadConverter)
+
+The order in which the Payload Converters are applied is important because during serialization, the Data Converter tries the Payload Converters in that specific order until a Payload Converter returns a non-nil payload.
```

Comment:
```suggestion
The order in which the Payload Converters are applied is important because during serialization the Data Converter tries the Payload Converters in that specific order until a Payload Converter returns a non-nil payload.
```